### DISCUSSION TITLE
How to validate React select field with react hook form

### DISCUSSION THREAD

**Discussion by apoorvmote** [react-hook-form/react-hook-form#5322] [Q&A] [answered]
I am using Creatable with React select.

How would I restrict the min length and max length of string from React select.

Please see my code sandbox link below 

https://codesandbox.io/s/react-hook-form-with-ui-library-forked-zp010?file=/src/index.js

The code sandbox is written in `JS` but in production I am using `Typescript` so please give types for validation also.

**Comment by bluebill1049** [comment, marked as answer]
use `validate` function to validate against that value as an object.
